fbw-a380x/src/systems/systems-host/systems/LegacyFuel.ts

```diff
@@ -127,6 +138,11 @@ export class LegacyFuel implements Instrument {
   }
 
   public onUpdate(): void {
+    const dt = this.sysHost.deltaTime;
+    if (!this.hasInit || !this.throttler.canUpdate(dt)) {
```

Comment:
canUpdate returns -1 if you can't update, and a cycle time > 0 if you can.

```suggestion
    if (!this.hasInit || this.throttler.canUpdate(dt) <= 0) {
```

Reply:
I like your suggestion better :D

Reply:
Haha :D I recently implemented your proposal, but forgot the minus, so I checked for ``=== 1``, which of course didn't work. That's why I'm now moving to >0 or <= 0

Reply:
good catch, that is me being unattentive and assuming :D the name made me think it was boolean. Though I wager it would still work :D

Reply:
Afaik negative numbers evaluate to true, so no it wouldn’t have worked 😄

Reply:
Yep :D https://developer.mozilla.org/en-US/docs/Glossary/Truthy